cassandra-analytics-core/src/main/java/org/apache/cassandra/spark/bulkwriter/CassandraBulkSourceRelation.java

```diff
@@ -69,7 +78,7 @@ public class CassandraBulkSourceRelation extends BaseRelation implements Inserta
     private final Broadcast<BulkWriterContext> broadcastContext;
     private final BulkWriteValidator writeValidator;
     private final SimpleTaskScheduler simpleTaskScheduler;
-    private volatile ImportCompletionCoordinator importCoordinator = null; // value is only set when using S3_COMPAT
+    private volatile ImportBarrier importBarrier = null; // value is only set when using S3_COMPAT
```

Comment:
does this need to be volatile? Do we have a different thread updating / reading this value?

Reply:
no. removed. Cannot remember why it was declared this way. (volatile is not added in this patch)